kumascript2/src/parser.pegjs

```diff
@@ -0,0 +1,135 @@
+/*
```

Comment:
The parser.js file is generated from this one. This is slightly changed from the old KumaScript to get better error reporting. And we now build parser.js at build time instead of at runtime.